### PR TITLE
rt-ide : Go to GitHub for TizenRT related guides

### PR DESCRIPTION
**Pull request by JeongJunSik**
- Word modification: Tizen RT -> TizenRT
- Remove smartthings-ext tab
- Add GitHub Link

### CODE REVIEW

File: docs/application/tizen-studio/rt-ide/overview.md [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
-	The Tizen RT Setup Wizard is a computer application for easily setting up Tizen IoT on hardware targets. In the Tizen OS version 4.0, flashing a platform image into ARTIK 053(s) is only supported with the IoT Setup Wizard in a Linux computer environment.
-
-To start developing your Tizen IoT applications:
-1. [Install the Tizen Studio for RT](getting-started/install.md)
-2. [Manage the project](getting-started/create.md)
-3. [Managing SmartThings&trade; Certificates](smartthings-ext/certificate-smartthings.md)
-4. [Flash the project](getting-started/flash.md)
-5. [Use the serial terminal](getting-started/terminal.md)
-6. [Debug the project](getting-started/debug.md)
+For more information on TizenRT, please refer to GitHub
```
Comment (mijin85cho) [inline docs/application/tizen-studio/rt-ide/overview.md]: How about changing line 15 and 16 into `For more information on TizenRT, please refer to [TizenRT GitHub](https://github.com/Samsung/TizenRT)`?

Comment (JeongJunSik) [reply docs/application/tizen-studio/rt-ide/overview.md]: Ok. i will change it.

Comment (mijin85cho) [reply docs/application/tizen-studio/rt-ide/overview.md]: In the updated version, we have `#` in front of `TizenRT GitHub`. If it is not necessary, how about deleting it?

image

**Comment by mijin85cho** [comment]
Hello, @annie-samsung

Please check this PR and refer that the TizenRT document structure will be changed to simply guide to TizenRT github link. If you just approve it, I'll merge it and apply the change to Tizen Developer Site.

Hello, @JeongJunSik,
I think I couldn't get the answer from you yet. Is it ok to use "#TizenRT GitHub"?


Regards

File: docs/application/tizen-studio/rt-ide/overview.md [hunk 2]
```diff
@@ -1,24 +1,15 @@
-# Tizen RT
+# TizenRT
 
-Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of Tizen RT is to extend the Tizen platform device coverage to these kind of low-end devices.
+Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
```
Comment (annie-samsung) [inline docs/application/tizen-studio/rt-ide/overview.md]: Suggest modifying first line to:

Tizen platform is commercialized on smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances.

Comment (annie-samsung) [reply docs/application/tizen-studio/rt-ide/overview.md]: A minor change (make TV and smartphone plural)...
Tizen platform is commercialized on smart TVs, smartphones, wearable devices (Gear S, Gear Fit), and smart home appliances.

**Comment by journey2w** [comment]
@JeongJunSik 
Please see our review comments and update your commit as our comments.

**Comment by mijin85cho** [comment]
Hello, @JeongJunSik
This is reminder of this PR. Would you please check the comments and update the PR?

File: docs/application/tizen-studio/rt-ide/overview.md (hunk 2 above)
Comment (JeongJunSik) [reply docs/application/tizen-studio/rt-ide/overview.md]: Ok. i will fix. Thanks.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/250

File: docs/application/tizen-studio/rt-ide/overview.md [hunk 3]
```diff
@@ -1,24 +1,15 @@
...
-	The Tizen RT Setup Wizard is a computer application for easily setting up Tizen IoT on hardware targets. In the Tizen OS version 4.0, flashing a platform image into ARTIK 053(s) is only supported with the IoT Setup Wizard in a Linux computer environment.
-
-To start developing your Tizen IoT applications:
-1. [Install the Tizen Studio for RT](getting-started/install.md)
-2. [Manage the project](getting-started/create.md)
-3. [Managing SmartThings&trade; Certificates](smartthings-ext/certificate-smartthings.md)
-4. [Flash the project](getting-started/flash.md)
-5. [Use the serial terminal](getting-started/terminal.md)
-6. [Debug the project](getting-started/debug.md)
+For more information on TizenRT, please refer to [#TizenRT GitHub](https://github.com/Samsung/TizenRT)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/rt-ide/overview.md]: Why are you keeping this # in spite of the review comment? Is there any reason that we must keep this # in front of "TizenRT Github"?

File: docs/application/tizen-studio/rt-ide/overview.md [hunk 4]
```diff
@@ -1,24 +1,15 @@
-# Tizen RT
+# TizenRT
 
-Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of Tizen RT is to extend the Tizen platform device coverage to these kind of low-end devices.
+Tizen platform is commercialized on smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/rt-ide/overview.md]: Would you please re-check the update? That I can't see the fix in below communication.


image

File: docs/application/tizen-studio/rt-ide/overview.md (hunk 3 above)
Comment (JeongJunSik) [reply docs/application/tizen-studio/rt-ide/overview.md]: Sorry, this is my mistake.
I fixed.

File: docs/application/tizen-studio/rt-ide/overview.md (hunk 4 above)
Comment (JeongJunSik) [reply docs/application/tizen-studio/rt-ide/overview.md]: i fixed.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/250

**Comment by mijin85cho** [comment]
Hello, @annie-samsung.
This comment is to remind you this PR.
Would you please check the latest files?

**Comment by mijin85cho** [comment]
Hello, @annie-samsung 
They decided to lead users directly to [TizenRT GitHub](https://github.com/Samsung/TizenRT),
the repository for TizenRT.
They will manage the documents for TizenRT on that repository,
and that's why they removed duplicated docs on tizen-docs.

**Comment by annie-samsung** [comment]
Hello, @mijin85cho 
Agreed, but if you notice this page is landing page for RT IDE (Tizen Studio for RT), not landing page for TizenRT (OS).

We should have a separate landing page for TizenRT outside Tizen Studio (page menus).

Regards,
Annie 

image

**Comment by mijin85cho** [comment]
Hello, @annie-samsung and @JeongJunSik 
How about remaining the links to the getting started pages?
Then this page will be like below.

image


With this format, the page will have simple description for TizenRT, leading links to how to use RT IDE, and a link to TizenRT github for the case of "more information".

@JeongJunSik,
If you are going to remove the pages under /smartthings-ext,
please remove the smartthings-ext/media folder, too.
We have below images left. Why are they not deleted?
 - rt_acco_act.png
 - rt_acco_sig.png
 - rt_cer_mang.png
 - rt_cert_gen.png
 - rt_import.png
 - rt_import_cer.png
 - rt_plus.png
 - rt_tick.png

Also please check docs/application/tizen-studio/TOC.md file, too.
To remove the files, please remove the links in the TOC.md files, also.

**Comment by mijin85cho** [comment]
@JeongJunSik , To merge this PR, please resolve the conflict.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/250

**Comment by mijin85cho** [comment]
Based on the discussion with @annie-samsung, I merge this PR.
Thanks for the review and the update.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**